TombEngine/Game/animation.h

```diff
@@ -22,6 +22,16 @@ enum class AnimCommandType
 	Flipeffect
 };
 
+struct SparkOffsets
```

Comment:
This struct and the `GetSparkOffsets()` function that returns it aren't really necessary, there are simpler ways of doing this. We can discuss on Discord. :)

Reply:
I needed a container with 6 arguments

Reply:
I did a little tinkering, and in fact, it seems we don't even need the struct. We can do everything in `TriggerAttackSpark()`, which is very convenient. The steps are a bit much to convey through text so I will push something in 8 hours to show you. :)